### PR TITLE
Normalization benchmarks

### PR DESCRIPTION
**Pull request by Priya2698**
Adds `batchnorm` and `instancenorm` benchmarks (in training mode).
The fusion definitions were added manually since lightning does not currently support `batchnorm`and `instancenorm`.

Note: This PR is aimed at adding more benchmark fusion definitions and does not address the coverage of input sizes.
Edit: Removed `gelu` benchmarks, will add them in a separate PR.

### CODE REVIEW

**Comment by Priya2698** [comment]
Should be rebased on #1121 and merged after #1132.

File: python_benchmarks/global_params.py [hunk 1]
```diff
@@ -25,7 +41,7 @@ def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
 
 
 # Datatypes to benchmark
-FLOAT_DTYPES = [torch.bfloat16, torch.float16, torch.float32]
+FLOAT_DTYPES = [torch.float16, torch.float32]
```
Comment (naoyam) [inline python_benchmarks/global_params.py:44]: Why `bfloat16` is dropped?

Comment (Priya2698) [reply python_benchmarks/global_params.py:44]: It fails validation tests right now with the constant tolerances. So I dropped it for now, will add it back when I switch to `testValidate`.

Comment (naoyam) [reply python_benchmarks/global_params.py:44]: OK, that's fine but please leave a comment.

**Review by naoyam — APPROVED** [review]
LGTM

**Pull request merged**